inference-engine/src/mkldnn_plugin/nodes/mkldnn_reduce_node.cpp

```diff
@@ -360,18 +365,9 @@ struct jit_uni_reduce_kernel_f32 : public jit_uni_reduce_kernel, public jit_gene
 
                 // load
                 load_scalar(xmm_dst, ptr[reg_dst], jcp_.dst_dt);
```

Comment:
What do you think, can we use load/store emitters to process the tail at once without looping element by element? If so, is it worth the effort?

Reply:
I've used the load/store emitters in other layers, it's truely convenient to use! Yet the main concern is that, in reduce layer, there is 50% chance that the innermost(eg. W of ncsp layout, C of nspc and blocked layout) dimension will be reduced. In these cases, element by element calculation can not be avoided, which means load/store_scalar routinues still need to be reserved. Further discussions are always welcomed. Thanks Maksim!